src/EnergyPlus/RoomAirModelAirflowNetwork.cc

```diff
@@ -0,0 +1,1482 @@
+#include <ObjexxFCL/ArrayS.functions.hh>
+#include <ObjexxFCL/Array1D.hh>
+#include <ObjexxFCL/Fmath.hh>
+#include <ObjexxFCL/MArray.functions.hh>
+
+// EnergyPlus Headers
+#include <RoomAirModelAirflowNetwork.hh>
+// #include <DataInterfaces.hh>
+#include <BaseboardElectric.hh>
+#include <BaseboardRadiator.hh>
+#include <DataAirflowNetwork.hh>
+#include <DataDefineEquip.hh>
+#include <DataEnvironment.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalFanSys.hh>
+#include <DataHeatBalance.hh>
+#include <DataHeatBalSurface.hh>
+#include <DataHVACGlobals.hh>
+#include <DataLoopNode.hh>
+#include <DataMoistureBalance.hh>
+#include <DataMoistureBalanceEMPD.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataRoomAirModel.hh>
+#include <DataSurfaces.hh>
+#include <DataSurfaceLists.hh>
+#include <DataZoneEquipment.hh>
+#include <ElectricBaseboardRadiator.hh>
+#include <General.hh>
+#include <HeatBalanceHAMTManager.hh>
+#include <HeatBalFiniteDiffManager.hh>
+#include <HighTempRadiantSystem.hh>
+#include <HWBaseboardRadiator.hh>
+#include <InternalHeatGains.hh>
+#include <InputProcessor.hh>
+#include <UtilityRoutines.hh>
+#include <FluidProperties.hh>
+#include <General.hh>
+#include <InputProcessor.hh>
+#include <InternalHeatGains.hh>
+#include <MoistureBalanceEMPDManager.hh>
+#include <OutputProcessor.hh>
+#include <OutputReportTabular.hh>
+#include <Psychrometrics.hh>
+#include <RefrigeratedCase.hh>
+#include <ScheduleManager.hh>
+#include <SteamBaseboardRadiator.hh>
+#include <UtilityRoutines.hh>
+#include <ZoneDehumidifier.hh>
+#include <ZonePlenum.hh>
+
+namespace EnergyPlus {
+
+namespace RoomAirModelAirflowNetwork {
+
+	// MODULE INFORMATION:
+	//       AUTHOR         Brent Griffith
+	//       DATE WRITTEN   November 2009
+	//       MODIFIED       na
+	//       RE-ENGINEERED  na
+
+	// PURPOSE OF THIS MODULE:
+	// contains the RoomAir model portions of RoomAirflowNetwork modeling
+
+	// METHODOLOGY EMPLOYED:
+	// Interact with Surface HB and Airflow Network Domains to get
+	// inputs.Do heat balance calculations on roomair nodes.
+
+	// REFERENCES:
+	// none
+
+	// OTHER NOTES:
+	// na
+
+	// Using/Aliasing
+	using namespace DataPrecisionGlobals;
+	using DataGlobals::MaxNameLength;
+	using DataGlobals::NumOfZones;
+	using namespace DataRoomAirModel;
+	using namespace DataHeatBalSurface;
+	using namespace DataMoistureBalance;
+	using namespace DataSurfaces;
+	using namespace DataHeatBalance;
+	// Data
+	// MODULE PARAMETER DEFINITIONS:
+
+	// MODULE DERIVED TYPE DEFINITIONS:
+
+	// INTERFACE BLOCK SPECIFICATIONS:
+	// na
+
+	// MODULE VARIABLE DECLARATIONS:
+	// see DataRoomAir
+
+	// SUBROUTINE SPECIFICATIONS FOR MODULE 
+
+	// Object Data
+	Array1D< RAFNData > RAFN;
+
+	// Functions
+
+	void
+	SimRoomAirModelAirflowNetwork( int const ZoneNum ) // index number for the specified zone
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Brent Griffith
+		//       DATE WRITTEN   January 2004/Aug 2005
```

Comment:
@lgu1234 I'm pretty sure that this subroutine information comment is out of date, so it either needs to be revised or removed. The same goes for all of the other info comments. It is fine with me if you just remove all of these comments from the new functions. The history is all contained in git, so... I guess I would remove everything but the "purpose" comment.

But you and/or others may have a different opinion, so I'll leave it up to you (@mjwitte @Myoldmopar ping).